Automattic/vip-go-mu-plugins
# Flush cache for page: also purge the AMP url

## Description

One of the feature requests for the Flush Cache for Page button is to also flush the AMP version.

## Changelog Description

### Plugin Updated: VIP Cache Manager

The Flush Cache For Page admin bar button now also purges AMP URL.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out.
2. Log in to a test site
3. Open the Network tab of browser dev tools
4. Click the flush button
5. Verify the payload contains the AMP version.
